test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextBuilderTests.cs

```diff
@@ -176,6 +178,9 @@ public void FillsRuntimeLibraryProperties()
             lib.Dependencies.Should().OnlyContain(l => l.Name == "System.Collections" && l.Version == "3.3.3");
             lib.ResourceAssemblies.Should().OnlyContain(l => l.Path == "en-US/Pack.Age.resources.dll" && l.Locale == "en-US");
 
+            // When ProjectModel supports path in the lock file library, this should assert "path/TO/package".
```

Comment:
The `PackageDescription` test helper method in this test suite initializes a `PackageDescription` type, which lives in ProjectModel. Until the `PackageDescription` type has a property to contain the relative path found in the lock file, it can't be plumbed through to the `.deps.json`, right? For example, in [CLI's `DependencyContextBuilder`](https://github.com/dotnet/cli/blob/49527d3ea6f26f2cfe5ac1414ff6c9d4ef0417a0/src/Microsoft.DotNet.ProjectModel/DependencyContextBuilder.cs#L140-L150), we create a runtime library out of a `PackageDescription` (well, `LibraryDescription` in this case).